GP02HYPM/GP02HYPM_D00008_ingest.csv

```diff
@@ -0,0 +1,13 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+#mi.dataset.driver.sio_eng.sio.sio_eng_sio_telemetered_driver,/omc_data/whoi/OMC/GP02HYPM/D00008/*status*.dat,GP02HYPM-RIM01-00-SIOENG000,telemetered,Not Expected,No telemetered data expected (no gliders deployed)
+#mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_sio_telemetered_driver,/omc_data/whoi/OMC/GP02HYPM/D00008/*ctdmo*.dat,GP02HYPM-RIM01-02-CTDMOG039,telemetered,Not Expected,No telemetered data expected (no gliders deployed)
+#mi.dataset.driver.WFP_ENG.wfp_sio.wfp_eng_wfp_sio_telemetered_driver,/omc_data/whoi/OMC/GP02HYPM/D00008/*.we_wfp_1237121.dat,GP02HYPM-WFP02-00-WFPENG000,telemetered,Not Expected,No telemetered data expected (no gliders deployed)
+#mi.dataset.driver.flord_l_wfp.sio.flord_l_wfp_sio_telemetered_driver,/omc_data/whoi/OMC/GP02HYPM/D00008/*.we_wfp_1237121.dat,GP02HYPM-WFP02-01-FLORDL000,telemetered,Not Expected,No telemetered data expected (no gliders deployed)
+#mi.dataset.driver.dosta_ln.wfp_sio.dosta_ln_wfp_sio_telemetered_driver,/omc_data/whoi/OMC/GP02HYPM/D00008/*.we_wfp_1237121.dat,GP02HYPM-WFP02-03-DOSTAL000,telemetered,Not Expected,No telemetered data expected (no gliders deployed)
+#mi.dataset.driver.ctdpf_ckl.wfp_sio.ctdpf_ckl_wfp_sio_telemetered_driver,/omc_data/whoi/OMC/GP02HYPM/D00008/*.wc_wfp_1237121.dat,GP02HYPM-WFP02-04-CTDPFL000,telemetered,Not Expected,No telemetered data expected (no gliders deployed)
+#mi.dataset.driver.vel3d_l.wfp.sio.vel3d_l_wfp_sio_telemetered_driver,/omc_data/whoi/OMC/GP02HYPM/D00008/*.wa_wfp_1237121.dat,GP02HYPM-WFP02-05-VEL3DL000,telemetered,Not Expected,No telemetered data expected (no gliders deployed)
+#mi.dataset.driver.WFP_ENG.wfp_sio.wfp_eng_wfp_sio_telemetered_driver,/omc_data/whoi/OMC/GP02HYPM/D00008/*.we_wfp_1237122.dat,GP02HYPM-WFP03-00-WFPENG000,telemetered,Not Expected,No telemetered data expected (no gliders deployed)
+#mi.dataset.driver.flord_l_wfp.sio.flord_l_wfp_sio_telemetered_driver,/omc_data/whoi/OMC/GP02HYPM/D00008/*.we_wfp_1237122.dat,GP02HYPM-WFP03-01-FLORDL000,telemetered,Not Expected,No telemetered data expected (no gliders deployed)
+#mi.dataset.driver.dosta_ln.wfp_sio.dosta_ln_wfp_sio_telemetered_driver,/omc_data/whoi/OMC/GP02HYPM/D00008/*.we_wfp_1237122.dat,GP02HYPM-WFP03-03-DOSTAL000,telemetered,Not Expected,No telemetered data expected (no gliders deployed)
+#mi.dataset.driver.ctdpf_ckl.wfp_sio.ctdpf_ckl_wfp_sio_telemetered_driver,/omc_data/whoi/OMC/GP02HYPM/D00008/*.wc_wfp_1237122.dat,GP02HYPM-WFP03-04-CTDPFL000,telemetered,Not Expected,No telemetered data expected (no gliders deployed)
+#mi.dataset.driver.vel3d_l.wfp.sio.vel3d_l_wfp_sio_telemetered_driver,/omc_data/whoi/OMC/GP02HYPM/D00008/*.wa_wfp_1237122.dat,GP02HYPM-WFP03-05-VEL3DL000,telemetered,Not Expected,No telemetered data expected (no gliders deployed)
```

Comment:
Looks good.

Reply:
Looks good - I think.  Not quite sure what the 'we, wc, wa' in filename mask are or where they come from, but they look the same as past deploy ingest sheets.